softwareComponents/configuration/include/configuration/rofiworld.hpp

```diff
@@ -36,6 +36,11 @@ enum class ModuleType {
     Cube,
 };
 
+enum class CollisionModel {
```

Comment:
I feel like this shouldn't be here. You don't use it anywhere in the library.
If you want to define it in one place, I would create a collision.hpp and move the code there - including the switch that returns the type.